modules/geo-layers/src/mvt-layer/mvt-layer.js

```diff
@@ -1,9 +1,9 @@
 import {log} from '@deck.gl/core';
 import {Matrix4} from 'math.gl';
 import {MVTWorkerLoader} from '@loaders.gl/mvt';
-import {binaryToGeoJson} from '@loaders.gl/gis';
+import {binaryToGeoJson, getSingleFeature} from '@loaders.gl/gis';
```

Comment:
Regarding loaders API design: Is `getSingleFeature` basically `binaryToGeoJson` with an index? Why are they named completely different? Can `binaryToGeoJson` just handle this with an option?

Reply:
`binaryToGeoJson` already has a fairly complex signature, the `format` parameter is said to control the output, but in fact also places restrictions on the input. If `format` is `'geometry'` then it expects a single `BinaryGeometry` as input. In this case it makes no sense to pass an index to select a feature. On the other hand, if `format` is `'feature` it expects a collection of features as input, as `BinaryFeatures`. 

My view is that it would be cleaner to split this function into two, so that we don't have implicit assumptions. It would also help with regards to types being clear. I would propose:

- `binaryGeometryToGeoJson(geometry: BinaryGeometry): Geometry`
- `binaryFeaturesToGeoJson(features: BinaryFeatures, options:BinaryFeaturesToGeoJsonOptions = {}): Features[]`

Where `type BinaryFeaturesToGeoJsonOptions = {type?: BinaryGeometryType, globalFeatureId?: number}`

I recognize that this is breaking change, but otherwise I feel we are overloading the original function with too many options. As we are about to release 3.0 then it seems a good time to make a breaking change if needed.

CC @ibgreen @kylebarron

Reply:
Agreed. We recently added types to these functions inside loaders.gl and it was very confusing to say the least, so I am supportive of this change if we can move fast.

Since we are late in the cycle, I suppose we can export the two functions you suggest but also keep the original function call those functions, but add the `@deprecated Use ... instead` note in the JSDoc header.

Reply:
I've added back the original function with the @deprecation annotation

Reply:
I use `@deprecated`, and get warnings and vscode cross outs. Not sure if `@Deprecation` works similarly.

Reply:
Sorry, that is just a typo, my computer changed the spelling. I used `@deprecated` in the PR